### PR TITLE
Updates to about page 

### PR DESCRIPTION
**Pull request by ramyaragupathy**
Updating about page to reflect changes to Governing board

cc @wipfli 

image

### CODE REVIEW

**Review by nyurik — COMMENTED** [review]
thx for the PR!  The description says that it updates the board, but i only see changes related to the twitter re-branding. Am I missing something?

I am not sure we really want to follow Twitter's re-branding - far too many places still use the word twitter, whereas `x` is ambiguous, and not worth the change IMO.  I think communication with the word "Twitter" will get our point across faster :)

**Comment by ramyaragupathy** [comment]
Still in draft, making changes one by one.

**Comment by ramyaragupathy** [comment]
and the issue specified in the commit message is not helpful as it is pointing to a diffferent one in the public repo. I was referring to an it without realising it is in the internal tracker.

File: content/about/luke/_index.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ large organizations), and that's why I initially led (along with my fearless
 fellow founding members) the push to ensure there was a single,
 vendor-neutral, and community-led fork of Mapbox GL JS, and I'm really
 excited to see the community and engagement grow over the last years. So
-far, I've been most involved in the background, from Twitter-spamming at the
+far, I've been most involved in the background, from X-spamming at the
```
Comment (lseelenbinder) [inline content/about/luke/_index.md:23]: I would prefer this remains "Twitter"—it was Twitter at the time. 😄

Comment (ramyaragupathy) [reply content/about/luke/_index.md]: done

**Comment by ramyaragupathy** [comment]
@nyurik - please review

image

**Comment by ramyaragupathy** [comment]
@klokan -  please review, I would remove the globe viz here in this section

image

**Comment by ramyaragupathy** [comment]
@hy9be - please review the content, once you update your GitHub profile picture, it should automatically reflect here.

image

**Comment by ramyaragupathy** [comment]
@birkskyum - please review your update here

image

**Comment by ramyaragupathy** [comment]
@lseelenbinder - here goes the latest for your page

image

File: data/board.json [hunk 2]
```diff
@@ -5,9 +5,9 @@
     "avatar": "https://avatars.githubusercontent.com/u/74932975?v=4"
   },
   {
-    "name": "Harel Mazor",
-    "link": "harel",
-    "avatar": "https://avatars.githubusercontent.com/u/3269297?v=4"
+    "name": " Haowen You",
```
Comment (wipfli) [inline data/board.json:8]:
```suggestion
    "name": "Haowen You",
```

**Review by wipfli — APPROVED** [review]
Thanks @ramyaragupathy

**Pull request merged**